operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -172,6 +172,7 @@ mimir:
     blocks_storage:
       backend: s3
       bucket_store:
+        max_chunk_pool_bytes: 2147483648 # 12GiB
```

Comment:
Just curious, why is this necessary?

Reply:
This option was recently deprecated and removed in #4996 I think it should be removed from this PR too

Reply:
@Logiraptor  there's a step in the pipeline that compares jsonnet values to the helm values 
https://github.com/grafana/mimir/actions/runs/5090989185/jobs/9150476748?pr=5081

@dimitarvdimitrov  I will do that thx